worldengine/generation.py

```diff
@@ -130,11 +130,32 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
             ('plain', hl),
             ('hill', ml),
             ('mountain', None)]
+    armonize_ocean(ocean, e, ocean_level)
     world.set_ocean(ocean)
     world.set_elevation(e, e_th)
     world.sea_depth = sea_depth(world, ocean_level)
 
 
+def armonize_ocean(ocean, elevation, ocean_level):
+    """
+    The goal of this function is to make the ocean floor less noisy.
+    The underwater erosion should cause the ocean floor to be more uniform
+    """
+    width, height = elevation.shape
+
+    shallow_sea = ocean_level * 0.85
```

Comment:
ocean_level is interpreted as "height of the ocean", right? (whereas platec uses "surface covered by ocean")
Can ocean_level be negative and throw strange results at this point? It depends on whether platec can return heightmaps with values below zero, I guess.